tethys_apps/base/url_map.py

```diff
@@ -17,7 +17,7 @@ class UrlMapBase:
 
     root_url = ''
 
-    def __init__(self, name, url, controller, protocol='http', regex=None):
+    def __init__(self, name, url, controller, protocol='http', regex=None, handler=None, handler_type=None):
         """
         Constructor
 
```

Comment:
This docstring needs to be updated with descriptions of the new `handler` and `handler_type` arguments. And `protocol` for that matter.